### PR TITLE
[MultilevelMonteCarloApplication] Removing compss library from the tests of the application

### PR DESCRIPTION
**Pull request by riccardotosi**
Removing compss library from the tests. Now the tests only need the mmg library installed.

### CODE REVIEW

**Comment by philbucher** [comment]
if you use the kratos testing-framework then you can skip tests in case e.g. some libraries or apps are not available

**Comment by riccardotosi** [comment]
@philbucher The point is that no one has now this library so it would never be run. This way the test can be run without problems.

**Comment by RiccardoRossi** [comment]
@riccardotosi still you should check if the mmg is there, for example by cheching it the MeshingApplication is available

File: applications/MultilevelMonteCarloApplication/tests/test_MultilevelMonteCarloApplication.py [hunk 1]
```diff
@@ -26,20 +27,19 @@ def AssembleTestSuites():
...
     nightSuite = suites['nightly']
-    nightSuite.addTest(KratosMultilevelMonteCarloGeneralTests('testNightlyFirstExample'))
-    nightSuite.addTest(KratosMultilevelMonteCarloGeneralTests('testNightlySecondExample'))
-    nightSuite.addTest(KratosMultilevelMonteCarloGeneralTests('testMonteCarloAnalysis'))
-    nightSuite.addTest(KratosMultilevelMonteCarloGeneralTests('testMultilevelMonteCarloAnalysis'))
+    if(hasattr(MeshingApplication,"MmgProcess2D")):
+        nightSuite.addTest(KratosMultilevelMonteCarloGeneralTests('testMonteCarloAnalysis'))
+        nightSuite.addTest(KratosMultilevelMonteCarloGeneralTests('testMultilevelMonteCarloAnalysis'))
+    else:
+        print("MMG process is not compiled and the corresponding tests will not be executed")
```
Comment (riccardotosi) [inline applications/MultilevelMonteCarloApplication/tests/test_MultilevelMonteCarloApplication.py:43]: @RiccardoRossi Here I do what you are saying

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**